File: C practical file/Q6.cpp
// Q6. WAP to use all Logical operators without using class in C++.

#include <iostream>
using namespace std;

int main()
{
    // Declare variables
    int num1, num2, result;

    // Get input from user
    cout << "Enter 2 numbers: ";
    cin >> num1 >> num2;

    // Clear screen
    system("clear");

    // Display the entered numbers
    cout << "Entered numbers are: " << num1 << " " << num2 << endl;

    // Logical operators
    cout << "Logical operators:" << endl;

    // AND operator
    result = (num1 > 5) && (num2 < 5);
    cout << "(" << num1 << " > 5) && (" << num2 << " < 5) : " << result << endl;

    // OR operator
    result = (num1 > 5) || (num2 < 5);
    cout << "(" << num1 << " > 5) || (" << num2 << " < 5) : " << result << endl;

    // NOT operator
    result = !(num1 > 5);
    cout << "!(" << num1 << " > 5) : " << result << endl;

    result = !(num2 < 5);
    cout << "!(" << num2 << " < 5) : " << result << endl;

    // Display note
    cout << "\n\n**Note : Here '0' = false, '1' = true **" << endl;

    return 0;
}

File: Java Practical file/Q6.java
// Q6. WAP to read three numbers from user and display the max no.

import java.util.Scanner;

class Q6 {
    public static void main(String[] args) {
        // Create a scanner object to get input from the user
        Scanner scr = new Scanner(System.in);
        // Declare three integer variables to hold user input
        int num1, num2, num3;
        System.out.print("Enter 3 numbers: ");
        // Read three integer values from the user
        num1 = scr.nextInt();
        num2 = scr.nextInt();
        num3 = scr.nextInt();

        // Find the greatest number among the three
        if ((num1 > num2) && (num1 > num3)) {
            System.out.println(num1 + " is the greatest.");
        } else if (num2 > num3) {
            System.out.println(num2 + " is the greatest.");
        } else {
            System.out.println(num3 + " is the greatest.");
        }
        // Close the scanner object to free up resources
        scr.close();
    }
}
